scripts/Packages/Package_BGA/ipc_test.yaml

```diff
@@ -0,0 +1,43 @@
+Dialog_VFBGA-86_6x6mm_Layout10x10_P0.55mm_Ball0.25mm_IPC:
```

Comment:
How about renaming all of them, to indicate what is tested.
Like 
* `test_only_ball_definition`
* `test_ball_and_pad`
* ...

And remove the stuff that is not needed for the test (datasheet, row_skip).
And add a file header saying what is tested and why.

Reply:
Followed the file name convention Rene used along with added comments to the file and streamlined as you mention.